<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialChooser</class>
 <widget class="QDialog" name="SerialChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="text">
        <string>Refresh List</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QPushButton" name="pushButtonUseSelected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connect to Selected</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButtonAbout">
       <property name="text">
        <string>About...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="7">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Detected Serial Ports</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="pushButtonFind">
       <property name="text">
        <string>Find My Quantum</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
